### ISSUE TITLE
[FEATURE} Clearer distinction between offensive and defensive Masteries under the equipped items

### ISSUE THREAD

**Issue by Sad-theFaceless** [Tmktahu/WakForge#141]
Random Masteries under the equipped items are displayed the same wether they're elemental or resistance masteries. It would be nice if something was made to distinguish them.
One solution would be to put a small sword: 
![571](https://github.com/Tmktahu/WakForge/assets/21340420/f3873569-b90d-4b19-bd09-bf0d440a2a39)
and a small shield: 
![520](https://github.com/Tmktahu/WakForge/assets/21340420/ff70223a-0cb0-49cc-9ec4-50ba9e2938a4)
next to them.

**Comment by Tmktahu** [comment]
Fair point. I'll see what I can do with the limited space there is. A small icon like that isn't a bad idea. I'll add it to the priority list.
